src/software/thunderscope/robot_diagnostics/drive_and_dribbler_widget.py

```diff
@@ -33,18 +33,11 @@ def __init__(self, proto_unix_io):
 
         # Initialize tab screen
         self.tabs = QTabWidget()
-        per_wheel_tab = QWidget()
         direct_velocity_tab = QWidget()
 
         # Add tabs
-        self.tabs.addTab(per_wheel_tab, "Direct Per-Wheel Control")
         self.tabs.addTab(direct_velocity_tab, "Direct Velocity Control")
```

Comment:
Curious why you removed the per wheel tab? I've never used it (didn't know we had it), but I guess it could be useful for Mech/Elec.

Reply:
That makes more sense.
@LiCody I think it might be worthwhile to add a task for supporting per wheel velocity in robot diagnostic. You could probably link to a branch which has this implementation in it, and note that ramping is required before it can be tested.

Reply:
created a ticket